cmd/environment.go

```diff
@@ -169,4 +277,14 @@ This returns a direct URL to the backup, this is a signed download link with a l
 func init() {
 	getCmd.AddCommand(getBackupCmd)
 	getBackupCmd.Flags().StringP("backup-id", "B", "", "The backup ID you want to restore")
+	updateEnvironmentCmd.Flags().String("deploy-base-ref", "", "Updates the deploy base ref for the selected environment")
+	updateEnvironmentCmd.Flags().String("deploy-head-ref", "", "Updates the deploy head ref for the selected environment")
+	updateEnvironmentCmd.Flags().String("deploy-title", "", "Updates the deploy title for the selected environment")
+	updateEnvironmentCmd.Flags().String("namespace", "", "Update the namespace for the selected environment")
+	updateEnvironmentCmd.Flags().String("route", "", "Update the route for the selected environment")
+	updateEnvironmentCmd.Flags().String("routes", "", "Update the routes for the selected environment")
+	updateEnvironmentCmd.Flags().UintVarP(&environmentAutoIdle, "auto-idle", "a", 1, "Auto idle setting of the environment")
```

Comment:
This doesn't work :( it always sets auto-idle to 1 if the user doesn't define this flag. Since this is a patching operation, if a user disabled idling, then ran another update env and didn't provide the auto-idle flag, it would enable idling again.

There are some possible ways to work around this though, using a helper that checks if the flag is actually provided, and then only setting the auto-idle value if the flag is provided, example: https://gist.github.com/shreddedbacon/4a1f6e8b3c63f5ce4a0e2a887b41e324